src/main/java/org/kuali/kfs/pdp/businessobject/PaymentDetail.java

```diff
@@ -137,6 +137,18 @@ public boolean isDisbursementActionAllowed() {
     }
 
     /**
+     * Wrapper for addNote, loops through the list of notes passed in and calls that.
+     *
+     * @param pnts list of notes to add
+     */
+    public void addNotes(List<PaymentNoteText> pnts) {
+        pnts.stream().forEach(n -> {
+            LOG.debug("addNotes() Creating check stub text note: " + n.getCustomerNoteText());
```

Comment:
Since this is doing a string concatenation, should this be in an isDebugEnabled if block?

Reply:
This is how the line appears in the base PaymentDetail file, but we can add the extra check if you think it's needed. (I don't think the "getCustomerNoteText" call is a very expensive operation anyway.)

Reply:
lets leave as is